.circleci/config.yml

```diff
@@ -551,282 +557,282 @@ workflows:
+          image_tag: '12.9'
           make: check-multi
           requires: [build-12]
       - test-citus:
           name: 'test-12_check-multi-1'
           pg_major: 12
-          image_tag: '12.8'
+          image_tag: '12.9'
           make: check-multi-1
           requires: [build-12]
       - test-citus:
           name: 'test-12_check-mx'
           pg_major: 12
-          image_tag: '12.8'
+          image_tag: '12.9'
           make: check-multi-mx
           requires: [build-12]
       - test-citus:
           name: 'test-12_check-vanilla'
           pg_major: 12
-          image_tag: '12.8'
+          image_tag: '12.9'
           make: check-vanilla
           requires: [build-12]
       - test-citus:
           name: 'test-12_check-isolation'
           pg_major: 12
-          image_tag: '12.8'
+          image_tag: '12.9'
           make: check-isolation
           requires: [build-12]
       - test-citus:
           name: 'test-12_check-worker'
           pg_major: 12
-          image_tag: '12.8'
+          image_tag: '12.9'
           make: check-worker
           requires: [build-12]
       - test-citus:
           name: 'test-12_check-operations'
           pg_major: 12
-          image_tag: '12.8'
+          image_tag: '12.9'
           make: check-operations
           requires: [build-12]
       - test-citus:
           name: 'test-12_check-follower-cluster'
           pg_major: 12
-          image_tag: '12.8'
+          image_tag: '12.9'
           make: check-follower-cluster
           requires: [build-12]
       - test-citus:
           name: 'test-12_check-columnar'
           pg_major: 12
-          image_tag: '12.8'
+          image_tag: '12.9'
           make: check-columnar
           requires: [build-12]
       - test-citus:
           name: 'test-12_check-columnar-isolation'
           pg_major: 12
-          image_tag: '12.8'
+          image_tag: '12.9'
           make: check-columnar-isolation
           requires: [build-12]
       - tap-test-citus:
           name: 'test_12_tap-recovery'
           pg_major: 12
-          image_tag: '12.8'
+          image_tag: '12.9'
           suite: recovery
           requires: [build-12]
       - test-citus:
           name: 'test-12_check-failure'
           pg_major: 12
           image: citus/failtester
-          image_tag: '12.8'
+          image_tag: '12.9'
           make: check-failure
           requires: [build-12]
 
       - test-citus:
           name: 'test-13_check-multi'
           pg_major: 13
-          image_tag: '13.4'
+          image_tag: '13.5'
           make: check-multi
           requires: [build-13]
       - test-citus:
           name: 'test-13_check-multi-1'
           pg_major: 13
-          image_tag: '13.4'
+          image_tag: '13.5'
           make: check-multi-1
           requires: [build-13]
       - test-citus:
           name: 'test-13_check-mx'
           pg_major: 13
-          image_tag: '13.4'
+          image_tag: '13.5'
```

Comment:
I am wondering if we should add these versions to `parameters` section so that we have a single place to update them, for example we could have 3 different parameters such as:

```
pg_12:
   type: string
   default: 12.9
pg_13:
   type: string
   default: 13.5
pg_14:
   type: string
   default: 14.1
pg_upgrade:
   type: string
   default: 12.9-13.5-14.1
```

Then when updating new images, we only need to update them in the parameters section

Reply:
Would love this, but lets work on this separately from upgrading image versions and not block this PR on such a change.

Reply:
I tried it but it seems that we can not use parameters under workflow context. And I tried to parameterize image_tag but I couldn't find a method to parameterize it